### PR TITLE
Issue #1 start the stop watch

### PR DESCRIPTION
**Pull request by seanmurakami**
1 issue - stop-watch-start

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+let $startbutton = document.querySelector('button')
+let $changeddigit = document.querySelector('p')
+
+let x = function update() {
```
Comment (thebearingedge) [inline main.js]: Avoid using single-letter variable names. You named your function, just use that.

```js
function update() {

}

function showNumbber() {
  setInterval(update, 1000)
}

$startButton.addEventListener('click', showNumber)
```

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+  background-color: white;
+  color: rgb(60, 122, 60);
+  padding: 0;
```
Comment (thebearingedge) [inline styles.css]: Do `padding` and `background-color` really do anything for you here? Check all of your CSS properties to confirm that none of them are extraneous.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  text-align: center;
+}
+
+h1 {
+  padding-top: 3rem;
+  font-size: 4rem;
+}
+
+p {
+  font-size: 4rem;
```
Comment (thebearingedge) [inline styles.css]: Try to group your common typography properties.

```css
h1,
p {
  font-size: 4rem;
}

h1 {
  padding-top: 3rem;
}

p {
  padding: 1rem 0;
}
```

File: main.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+let $startbutton = document.querySelector('button')
```
Comment (thebearingedge) [inline main.js]: Stick with `camelCase` for JavaScript variable names.

```js
var $startButton
var $changedDigit
```

**Comment by thebearingedge** [comment]
Please reopen when ready.

**Comment by seanmurakami** [comment]
1 issue - stop-watch-start

**Pull request merged**